### PR TITLE
docs: adding an etcd v2 landing page.

### PR DESCRIPTION
**Pull request by zbwright**
Adding a page for the etcd v2 docs that can be linked from docs.md for the etcd v3 docs.

### CODE REVIEW

File: Documentation/v2/docs-v2.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Documentation
+
+etcd is a distributed key-value store designed to reliably and quickly preserve and provide access to critical data. It enables reliable distributed coordination through distributed locking, leader elections, and write barriers. An etcd cluster is intended for high availability and permanent data storage and retrieval.
+
+This is the etcd v2 documentation set. For more recent versions, please see the [etcd v3 guides][../docs.md].
```
Comment (heyitsanthony) [inline Documentation/v2/docs-v2.md]: `[etcd v3 guides](../docs.md)`

**Comment by zbwright** [comment]
replacing /v2/README.md works for me.

tell me, please: what does 'doc is dangling' mean?

**Comment by heyitsanthony** [comment]
@zbwright only in repo, not connected to other docs

**Comment by zbwright** [comment]
replaced by https://github.com/coreos/etcd/pull/8321. closing.

**Pull request closed without merging**